docker/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+RADICALE_UID="${RADICALE_UID:-1000}"
+RADICALE_GID="${RADICALE_GID:-1000}"
+
+getent group  radicale &> /dev/null || addgroup -g ${RADICALE_GID} radicale
+getent passwd radicale &> /dev/null || adduser -D -H -s /bin/false -u ${RADICALE_GID} -G radicale radicale
+
+if [ -n "${RADICALE_CONFIG}" ]; then
+    chown -R radicale:radicale $(dirname $RADICALE_CONFIG) /var/lib/radicale
+    CONF_OPT="--config '${RADICALE_CONFIG}'"
+else
+    chown -R radicale:radicale /etc/radicale /var/lib/radicale
+    CONF_OPT=""
+fi
```

Comment:
why don't you do all that user stuff in the docker file itself?

Reply:
because the variable's values are to be configured per container.